tools/accuracy_checker/accuracy_checker/adapters/mask_rcnn.py

```diff
@@ -152,12 +153,31 @@ def _process_tf_obj_detection_api_outputs(self, raw_outputs, identifiers, frame_
             return results
 
     def _process_pytorch_outputs(self, raw_outputs, identifiers, frame_meta):
-        classes = raw_outputs[self.classes_out]
+        classes = raw_outputs.get(self.classes_out, None)
+        boxes = raw_outputs.get(self.boxes_out, None)
+        scores = raw_outputs.get(self.scores_out, None)
+        raw_masks = raw_outputs.get(self.raw_masks_out, None)
+
+        if boxes is None:
+            for v in raw_outputs.values():
```

Comment:
It is not safe from quantization perspective (outputs dictionary extended with layers for statistics collection), I can not guarantee that in the model always only 2 layer with such shape and dims...

Reply:
I see. Changed the logic a bit, please take a look.